runtimes/runtimes/fqn/node/fqnWorkerPool.ts

```diff
@@ -0,0 +1,9 @@
+import path = require('path')
+import { CommonFqnWorkerPool } from '../common/commonFqnWorkerPool'
+import { WorkerPoolConfig } from '../../../server-interface'
+
+export class FqnWorkerPool extends CommonFqnWorkerPool {
```

Comment:
why is a worker pool needed? it might be obvious now, but others working on this project in the future won't have the context that you have 

a brief docstring on this class can give insight.